packages/tracking/src/types.ts

```diff
@@ -229,3 +232,8 @@ export declare type PagePathVisitedData = BaseEventData & {
   path_id: string;
   path_full_title: string;
 };
+
+export declare type CoursePageVisitedData = BaseEventData & {
```

Comment:
Oh I just noticed the `declare` above. It's redundant; these types are already being "declared".

```suggestion
export type CoursePageVisitedData = BaseEventData & {
```

Reply:
oh lol copy pasta ftw